packages/office-ui-fabric-react/src/components/ContextualMenu/examples/ContextualMenu.Icon.Example.tsx

```diff
@@ -1,106 +1,93 @@
+      iconProps: {
+        iconName: 'Share'
+      },
+      text: 'Share'
+    },
+    {
+      key: 'print',
+      iconProps: {
+        iconName: 'Print'
+      },
+      text: 'Print'
+    },
+    {
+      key: 'music',
+      iconProps: {
+        iconName: 'MusicInCollectionFill'
+      },
+      text: 'Music'
+    }
+  ]);
 
-    return (
-      <div>
-        <DefaultButton
-          text="Click for ContextualMenu"
-          menuProps={{
-            shouldFocusOnMount: true,
-            items: [
-              {
-                key: 'openInWord',
-                text: 'Open in Word',
-                onRenderIcon: (props: IContextualMenuItemProps) => {
-                  return (
-                    <span className={styles.iconContainer}>
-                      <Icon iconName={'WordLogoFill16'} className={styles.logoFillIcon} />
-                      <Icon iconName={'WordLogo16'} className={styles.logoIcon} />
-                    </span>
-                  );
-                }
-              },
-              {
-                key: 'newItem',
-                iconProps: {
-                  iconName: 'Add'
-                },
-                text: 'New'
-              },
-              {
-                key: 'upload',
-                onClick: () => {
-                  this.setState({ showCallout: true });
-                },
-                iconProps: {
-                  iconName: 'Upload',
-                  style: {
-                    color: 'salmon'
-                  }
-                },
-                text: 'Upload (Click for popup)',
-                title: 'Upload a file'
-              },
-              {
-                key: 'divider_1',
-                itemType: ContextualMenuItemType.Divider
-              },
-              {
-                key: 'share',
-                iconProps: {
-                  iconName: 'Share'
-                },
-                text: 'Share'
-              },
-              {
-                key: 'print',
-                iconProps: {
-                  iconName: 'Print'
-                },
-                text: 'Print'
-              },
-              {
-                key: 'music',
-                iconProps: {
-                  iconName: 'MusicInCollectionFill'
-                },
-                text: 'Music'
-              }
-            ]
-          }}
-        />
-        {showCallout && (
-          <Callout
-            setInitialFocus={true}
-            // tslint:disable-next-line:jsx-no-lambda
-            onDismiss={() => this.setState({ showCallout: false })}
-          >
-            <DefaultButton
-              // tslint:disable-next-line:jsx-no-lambda
-              onClick={() => this.setState({ showCallout: false })}
-              text="Hello Popup"
-            />
-          </Callout>
-        )}
-      </div>
-    );
-  }
-}
+  return (
+    <div>
+      <DefaultButton
+        text="Click for ContextualMenu"
+        menuProps={{
```

Comment:
useConst (in this case you could actually get rid of the separate const for the menu items, and maybe even onShowCallout, since they're only used within the menu props)